packages/compose/src/higher-order/with-is-connected/index.native.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * WordPress dependencies
+ */
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { useIsConnected } from '@wordpress/react-native-bridge';
+
+const withIsConnected = createHigherOrderComponent( ( WrappedComponent ) => {
+	return ( props ) => {
+		const isConnected = useIsConnected();
```

Comment:
Noting that this needed to be destructured. I'll fix it in a commit. 
```suggestion
		const { isConnected } = useIsConnected();
```